packages/synapse-react-client/src/components/styled/StyledTableContainer.tsx

```diff
@@ -1,64 +1,80 @@
-import { Box, BoxProps, styled } from '@mui/material'
 import { StyledComponent } from '@emotion/styled'
+import { Box, BoxProps, styled } from '@mui/material'
 
 export type StyledTableContainerProps = BoxProps & {
+  /**
+   * Controls the padding of the table cells
+   * @default 'default'
+   */
   density?: 'default' | 'compact'
+  /**
+   * Controls whether the rows of the table are striped using CSS. For virtualized tables, this should be set to false, since the
+   * row even/oddness is not guaranteed to be consistent as DOM nodes are loaded and unloaded.
+   * @default false
+   */
+  noStripedRows?: boolean
```

Comment:
noStripedRows should default to false for virtualized tables (where the row even/odd is not guaranteed)?  I would expect false if this was "stripedRows" in this case, or true for noStripedRows.  Is my understanding flawed, or is the logic flipped in the field name?

Reply:
This is the shared `StyledTableContainer`. Most of our tables are not virtualized, so having it striped by default is the most typical case. The virtual table component can set this to true